README.md

```diff
@@ -1,55 +1,93 @@
+Sia is a new decentralized cloud storage platform that radically alters the
+landscape of cloud storage. By leveraging smart contracts, client-side
+encryption, and sophisticated redundancy (via Reed-Solomon codes), Sia allows
+users to safely store their data with hosts that they do not know or trust.
+The result is a cloud storage marketplace where hosts compete to offer the
+best service at the lowest price. And since there is no barrier to entry for
+hosts, anyone with spare storage capacity can join the network and start
+making money.
+
+Traditional cloud storage has a number of shortcomings. Users are limited to a
+few big-name offerings: Google, Microsoft, Amazon. These companies have little
+incentive to encrypt your data or make it easy to switch services later. Their
+code is closed-source, and they can lock you out of your account at any time.
+
+We believe that users should own their data. Sia achieves this by replacing
+the traditional monolithic cloud storage provider with a blockchain and a
+swarm of hosts, each of which stores an encrypted fragment of your data. Since
+the fragments are redundant, no single host can hold your data hostage: if
+they jack up their price or go offline, you can simply download from a
+different host. In other words, trust is removed from the equation, and
+switching to a different host is painless. Stripped of these unfair
+advantages, hosts must compete solely on the quality and price of the storage
+they provide.
+
+Sia can serve as a replacement for personal backups, bulk archiving, content
+distribution, and more. For developers, Sia is a low-cost alternative to
+Amazon S3. Storage on Sia is a full order of magnitude cheaper than on S3,
+with comparable bandwidth, latency, and durability. Sia works best for static
+content, especially media like videos, music, and photos.
+
+Distributing data across many hosts automatically confers several advantages.
+The most obvious is that, just like BitTorrent, uploads and downloads are
+highly parallel. Given enough hosts, Sia can saturate your bandwidth. Another
+advantage is that your data is spread across a wide geographic area, reducing
+latency and safeguarding your data against a range of attacks.
+
+It is important to note that users have full control over which hosts they
+use. You can tailor your host set for minimum latency, lowest price, widest
+geographic coverage, or even a strict whitelist of IP addresses or public
+keys.
+
+At the core of Sia is a blockchain that closely resembles Bitcoin.
+Transactions are conducted in Siacoin, a cryptocurrency. The blockchain is
+what allows Sia to enforce its smart contracts without relying on centralized
+authority. To acquire siacoins, use an exchange such as [Poloniex](https://poloniex.com), [Yunbi](https://yunbi.com), or
+[Bitsquare](https://bitsquare.io).
+
+To get started with Sia, check out the guides below:
+
+- [How to Store Data on Sia](tbd)
```

Comment:
ah man the build passed too.

link to a private gist containing john's post, we'll update it to medium shortly after its published properly.